config/deploy/stage.rb

```diff
@@ -1,4 +1,9 @@
-server 'sul-lyberservices-test.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
+server 'sul-preassembly-stage.stanford.edu', user: 'preassembly', roles: %w{web app db}
 
 Capistrano::OneTimeKey.generate_one_time_key!
 set :rails_env, "test"
```

Comment:
can we make this be 'production' too?

Reply:
The reason being that I'd like to test the code as close to as we can to how we do it in production.  Only the shared configs should change.

Reply:
Done!